balancer/nginx.conf

```diff
@@ -0,0 +1,51 @@
+worker_processes  auto;
+
+error_log  /var/log/nginx/error.log warn;
```

Comment:
Logi do pliku? Nie znam waszej usługi, ale czy to na pewno najlepsze rozwiązanie?

Reply:
Balancer nie działa w usłudze HyperOne Strona, a w kontenerze w HyperOne Wirtualna Maszyna, gdyż HyperOne nie ma jeszcze usługi load-balancera. Tak, bo to jest symlink na /dev/stderr i opiera się na rekomendowanej konfiguracji nginx w Dockerze:
```
$ docker run nginx grep '' /etc/nginx/nginx.conf /etc/nginx/nginx/conf.d/ -R
grep: /etc/nginx/nginx/conf.d/: No such file or directory
/etc/nginx/nginx.conf:
/etc/nginx/nginx.conf:user  nginx;
/etc/nginx/nginx.conf:worker_processes  1;
/etc/nginx/nginx.conf:
/etc/nginx/nginx.conf:error_log  /var/log/nginx/error.log warn;
/etc/nginx/nginx.conf:pid        /var/run/nginx.pid;
/etc/nginx/nginx.conf:
/etc/nginx/nginx.conf:
/etc/nginx/nginx.conf:events {
/etc/nginx/nginx.conf:    worker_connections  1024;
/etc/nginx/nginx.conf:}
/etc/nginx/nginx.conf:
/etc/nginx/nginx.conf:
/etc/nginx/nginx.conf:http {
/etc/nginx/nginx.conf:    include       /etc/nginx/mime.types;
/etc/nginx/nginx.conf:    default_type  application/octet-stream;
/etc/nginx/nginx.conf:
/etc/nginx/nginx.conf:    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
/etc/nginx/nginx.conf:                      '$status $body_bytes_sent "$http_referer" '
/etc/nginx/nginx.conf:                      '"$http_user_agent" "$http_x_forwarded_for"';
/etc/nginx/nginx.conf:
/etc/nginx/nginx.conf:    access_log  /var/log/nginx/access.log  main;
/etc/nginx/nginx.conf:
/etc/nginx/nginx.conf:    sendfile        on;
/etc/nginx/nginx.conf:    #tcp_nopush     on;
/etc/nginx/nginx.conf:
/etc/nginx/nginx.conf:    keepalive_timeout  65;
/etc/nginx/nginx.conf:
/etc/nginx/nginx.conf:    #gzip  on;
/etc/nginx/nginx.conf:
/etc/nginx/nginx.conf:    include /etc/nginx/conf.d/*.conf;
/etc/nginx/nginx.conf:}
```